### ISSUE TITLE
Incorrect Formatting

### ISSUE THREAD

**Issue by rehatkathuria** [nicklockwood/SwiftFormat#1816]
The following:
```
public extension [
	Operations.GetConversationsQuery.Data.GetConversation
		.Asset?
] {
```

gets transformed incorrectly to the following:
```
public extension [
	Operations.GetConversationsQuery.Data.GetConversation
		.Asset?,
] {
```

with the following:
```
# General Options
--swiftversion 5.9

# Format Options
--allman false
--binarygrouping none
--closingparen balanced
--commas always
--conflictmarkers reject
--elseposition next-line
--exponentcase lowercase
--exponentgrouping disabled
--extensionacl on-extension
--fractiongrouping disabled
--fragment false
--header ignore
--hexgrouping none
--hexliteralcase uppercase
--ifdef indent
--importgrouping alphabetized
--indent tab
--indentcase false
--extensionacl on-extension
--trimwhitespace always
--maxwidth 80
--nospaceoperators ..<,...
--octalgrouping none
--operatorfunc no-space
--selfrequired
--stripunusedargs closure-only
--trimwhitespace always
--wraparguments before-first
--wrapcollections before-first
--wrapparameters before-first

# Rules
--disable hoistPatternLet
--disable redundantReturn
--disable redundantSelf
--disable trailingclosures
--disable wrapMultilineStatementBraces
--disable preferKeyPath
--disable hoistAwait

# Exclusions
--exclude **/*Generated/*
--exclude **/*generated*
--exclude vendor/*
--exclude .derived/*
```

**Comment by rehatkathuria** [comment]
I believe this is because of the commas always rule presuming that an array has been declared?

**Comment by nicklockwood** [comment]
@rehatkathuria fixed in 0.54.4

**Comment by rehatkathuria** [comment]
Thank you again for such a quick fix @nicklockwood! :)
